libxrdp/xrdp_sec.c

```diff
@@ -778,20 +778,20 @@ xrdp_sec_process_logon_info(struct xrdp_sec *self, struct stream *s)
         return 1;
     }
 
-    if (unicode_in(s, len_domain, self->rdp_layer->client_info.domain, 255) != 0)
+    if (unicode_in(s, len_domain, 512, self->rdp_layer->client_info.domain, 255) != 0)
```

Comment:
I don't think there is special meaning to define it, the values changes for every field, that is given in the protocol specs.